source/extensions/filters/http/ext_proc/ext_proc.cc

```diff
@@ -756,8 +752,8 @@ void Filter::addAttributes(ProcessorState& state, ProcessingRequest& req) {
   }
 
   auto activation_ptr = Filters::Common::Expr::createActivation(
-      &config_->expressionManager().localInfo(), state.callbacks()->streamInfo(),
-      state.requestHeaders(), dynamic_cast<const Http::ResponseHeaderMap*>(state.responseHeaders()),
+      state.callbacks()->streamInfo(), state.requestHeaders(),
```

Comment:
Did the function signature of `createActivation` change in upstream?

Reply:
Ah, I see. Are we currently tracking changes that we know we'll have to make when we bump to 1.29? If not, do you think it makes sense to leave an inline comment including a link to that PR/description of what changes will be necessary? Or will that just make things harder when we have to get the 1.29 bump out?

Reply:
I think it would be simpler to make the 1.29 changes from a 1.30-dev cherry-pick (which is what I did here), and since that release was cut pmuch right before these merged, these divergences won't be an issue.

Basically, if we are keel-hauling these changes forward through versions, we should track them, but if we cherry-pick from merged upstream it won't be an issue